content/en/docs/setup/platform-setup/openshift/index.md

```diff
@@ -34,22 +25,6 @@ After installation is complete, expose an OpenShift route for the ingress gatewa
 $ oc -n istio-system expose svc/istio-ingressgateway --port=http2
 {{< /text >}}
 
-## Privileged security context constraints for application sidecars
-
-The Istio sidecar injected into each application pod runs with user ID 1337, which is not allowed by default in OpenShift. To allow this user ID to be used, execute the following commands. Replace `<target-namespace>` with the appropriate namespace.
-
-{{< text bash >}}
-$ oc adm policy add-scc-to-group privileged system:serviceaccounts:<target-namespace>
-$ oc adm policy add-scc-to-group anyuid system:serviceaccounts:<target-namespace>
```

Comment:
Without this I get:
```
Error creating: pods "details-v1-79f774bdb9-" is forbidden: unable to validate against any security context constraint: [spec.containers[0].securityContext.runAsUser: Invalid value: 1000: must be in the ranges: [1000600000, 1000609999]]
```

Reply:
is there a difference between ocp 4.5 and 4.6? i verified that i dont have the namespaces added to the privileged nor the anyuid groups and my pods are starting without that error you have listed above. i dont have the option yet to upgrade to ocp 4.6.

Reply:
so maybe we just need the anyuid group still but the privileged group isn't needed anymore?

Reply:
yep, that's what I said below.